src/rpc/blockchain.cpp

```diff
@@ -781,10 +781,11 @@ static RPCHelpMan getblockfrompeer()
         "getblockfrompeer",
         "\nAttempt to fetch block from a given peer.\n"
         "\nWe must have the header for this block, e.g. using submitheader.\n"
-        "\nReturns {} if a block-request was successfully scheduled\n",
+        "Subsequent calls for the same block and a new peer will cause the response from the previous peer to be ignored.\n"
```

Comment:
Nit: This change appears to be part of the wrong commit

Reply:
Moved.